Libraries/CMSIS/Device/Maxim/MAX32572/Source/GCC/gcc.mk

```diff
@@ -34,407 +34,11 @@
  #
  ###############################################################################
 
-# The build directory
-ifeq "$(BUILD_DIR)" ""
-BUILD_DIR=$(CURDIR)/build
-endif
-
-# Create output object file names
-SRCS_NOPATH := $(foreach NAME,$(SRCS),$(basename $(notdir $(NAME))).c)
-OBJS_NOPATH := $(SRCS_NOPATH:.c=.o)
-OBJS        := $(OBJS_NOPATH:%.o=$(BUILD_DIR)/%.o)
-
-################################################################################
-# Goals
-
-# The default goal, which causes the example to be built.
-.DEFAULT_GOAL :=
-.PHONY: all
-all: mkbuildir
-all: ${BUILD_DIR}/${PROJECT}.elf
-
-# Goal to build for release without debug
-.PHONY: release
-release: mkbuildir
-release: ${BUILD_DIR}/${PROJECT}.elf
-release: ${BUILD_DIR}/${PROJECT}.srec
-release: ${BUILD_DIR}/${PROJECT}.hex
-release: ${BUILD_DIR}/${PROJECT}.bin
-release: ${BUILD_DIR}/${PROJECT}.dasm
-
-# The goal to build as a library
-.PHONY: lib
-lib: mkbuildir
-lib: ${BUILD_DIR}/${PROJECT}.a
-
-# The goal to create the target directory.
-.PHONY: mkbuildir
-mkbuildir:
-	@mkdir -p ${BUILD_DIR}
-
-# The goal to clean out all the build products.
-.PHONY: clean
-clean:
-	@rm -rf ${BUILD_DIR} ${wildcard *~}
-
-${BUILD_DIR}/${PROJECT}.elf: ${LIBS} ${OBJS} ${LINKERFILE}
-${BUILD_DIR}/${PROJECT}.a: ${OBJS}
-
-# Create a goal to exercise the library build dependencies
-.PHONY: FORCE
-FORCE:
+OLD_GCC_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
```

Comment:
This part hasn't been released yet so is support for old style projects necessary?

Reply:
Good point, I deleted the files in https://github.com/Analog-Devices-MSDK/msdk/pull/619/commits/e929295ea899f24c74efca7a1f9438b0db6b2235.  I also updated all the parts to point direct to the new locations.